packages.makefile

```diff
@@ -35,10 +35,10 @@ BUILDREQUIRESFEDORA := $(shell grep -s "FEDORA${OSVERSION}" SOURCES/.dependencie
 JAVAREQUIRES := $(shell grep -s "JAVA" SOURCES/.dependencies | cut -d ':' -f2)
 
 # Original URL: http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html#javasejdk
-JDKURL := http://www.normation.com/tarball/java/jdk-7u71-linux-i586.rpm
+JDKURL := http://www.normation.com/tarball/java/jdk-latest-linux-i586.rpm
```

Comment:
This looks like a symbolic URL that will provide a different version at different points of time. We must not use such "moving" versions, because it means we don't know how to rebuild a given release, which may make reproducing (and therefore fixing) bugs impossible.